runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3066,6 +3066,21 @@ gcInitializeDefaults(J9JavaVM* vm)
 
 	extensions->configuration = configurateGCWithPolicyAndOptions(vm->omrVM);
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/* Favour reduced memory consumption over pause times when checkpointing is enabled by
+	 * scaling the default min and max DNSS expected ratios by a constant factor, unless
+	 * at least one ratio was directly specified by the user.
+	 */
+	if (vm->internalVMFunctions->isCRaCorCRIUSupportEnabled(vm)) {
+		const double scaleFactor = 2;
+		if (!extensions->dnssExpectedRatioMaximum._wasSpecified &&
+		    !extensions->dnssExpectedRatioMinimum._wasSpecified) {
+			extensions->dnssExpectedRatioMaximum._valueSpecified *= scaleFactor;
+			extensions->dnssExpectedRatioMinimum._valueSpecified *= scaleFactor;
+		}
+	}
+#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+
```

Comment:
this is somewhat an ok place to deal with this,  butsince these fields are already initially set in OMR's MM_ConfigurationGenerational::initialize(), the matching place in J9 to override the values would be MM_ConfigurationDelegate::initialize()